includes/class-newspack-popups-inserter.php

```diff
@@ -88,7 +88,7 @@ public static function popups_for_post() {
 		$popups_to_maybe_display = array_filter(
 			$popups_to_maybe_display,
 			function( $popup ) {
-				return 'manual' !== $popup['options']['frequency'];
+				return ! Newspack_Popups_Custom_Placements::is_custom_placement( $popup );
```

Comment:
Since the custom placement is a `placement`, not a `frequency`, wouldn't the custom-placement prompts already be excluded in the return values of `retrieve_*_popups` methods? If so, then this filtering here would not be needed.

Reply:
True, removed in b8344de.